### PR TITLE
feat(database): adding stockage resource edition for managed

### PR DESCRIPTION
**Pull request by bdebon**
# What does this PR do?

[> Link to the JIRA ticket](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-681)
also this one : https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-674

Adding the possibility to set stockage for a managed db in creation and settings flow.

CleanShot 2023-04-12 at 13 39 07@2x

Also I added on this same PR the fact to show the banner on db edition:
CleanShot 2023-04-12 at 15 46 13@2x

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - feat(database): adding stockage resource edition for managed - 2023-04-12T11:40:21Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/4c3a0f83-d258-4b14-b4de-e69b5affb0cd/services/general)

 Another comment will be posted when deployments are terminated

**Comment by acarranoqovery** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://z4c3a0f83-zfc78ad5d-gtw.zc531a994.rustrocks.cloud)

File: libs/pages/database/src/lib/ui/page-settings-resources/page-settings-resources.tsx [hunk 1]
```diff
@@ -36,71 +37,79 @@ export function PageSettingsResources(props: PageSettingsResourcesProps) {
         <h2 className="h5 mb-8 text-text-700">Resources</h2>
         <form onSubmit={onSubmit}>
           <p className="text-text-500 text-xs mb-3">Manage the database's resources and horizontal auto-scaling.</p>
```
Comment (acarranoqovery) [inline libs/pages/database/src/lib/ui/page-settings-resources/page-settings-resources.tsx:39]: to update into: "Manage the database's resources"

**Comment by bdebon** [comment]
I still have to dig why the statuses are not fetched with a setInterval for dbs. It is the case for containers and application but not for db. As a result, we see the yellow banner flag only if we refresh or go back to the list of services.

File: libs/pages/services/src/lib/feature/page-database-create-feature/step-summary-feature/step-summary-feature.tsx [hunk 2]
```diff
@@ -33,11 +33,7 @@ export function StepSummaryFeature() {
   }
 
   const onBack = () => {
```
Comment (RemiBonnet) [inline libs/pages/services/src/lib/feature/page-database-create-feature/step-summary-feature/step-summary-feature.tsx:35]: You use only one time the onBack and gotoResources functions on this component, you can directly add them to the props for StepSummary

**Review by RemiBonnet — COMMENTED** [review]
Well done @bdebon, I added some comments!

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**